mangaki/mangaki/views.py

```diff
@@ -668,10 +669,13 @@ def get_reco_algo_list(request, algo_name, category):
     reco_list = []
     data = get_reco_algo(request, algo_name, category)
     works = data['works']
+    categories = dict(WORK_CATEGORY_CHOICES)
     for work_id in data['work_ids']:
         work = works[work_id]
-        reco_list.append({'id': work.id, 'title': work.title, 'poster': work.ext_poster, 'synopsis': work.synopsis,
-                          'category': work.category.slug})
+        reco_list.append({'id': work.id, 'title': work.title,
+                          'poster': work.ext_poster, 'synopsis': work.synopsis,
+                          'category_slug': work.category.slug,
+                          'category': str(categories[work.category.slug])})
```

Comment:
Hi @Elarnon, didn't see that you commented here.
The template needs `category_slug` for CSS and `category` for display in a potentially non-English language. That's why we need both.

Reply:
My question was if the template was already doing this but we were passing the wrong arguments, since the template was not modified by the commit.  I guess your answer is "yes", so LGTM.

Reply:
Now I remember! I fixed the problem for the mosaic and WorkList (API), but not the recommendations. That's why the template didn't need to be updated.